app/screens/people-screen/circles/helpers.ts

```diff
@@ -0,0 +1 @@
+export const getInviteLink = (username: string) => `https://get.blink.sv/${username}`
```

Comment:
should probably be in `app/config/appinfo.ts`

Reply:
done